src/AzureClient/AzureSubmissionContext.cs

```diff
@@ -37,6 +38,13 @@ public sealed class AzureSubmissionContext : IQuantumMachineSubmissionContext
         /// </summary>
         public string OperationName { get; set; } = string.Empty;
 
+        // NOTE: The following member is inherited from IQuantumMachineSubmissionContext, but it's not used.
+        //       This is supporting the change in https://github.com/microsoft/qsharp-runtime/pull/829
+        //       Notice that this is different from the existing 'InputParameters' member below.
+        //       Upcoming work will enable this functionality and align those similar named items.
+        /// <inheritdoc/>
+        public ImmutableDictionary<string, string> InputParams { get; set; } = new Dictionary<string, string>();
```

Comment:
It feels a little odd to declare this property as an immutable dictionary, then populate it with a mutable dictionary? I'm guessing that relies on an implicit operator, but it may be better to explicitly refer to the empty immutable dictionary:

```suggestion
        public ImmutableDictionary<string, string> InputParams { get; set; } = Dictionary<string, string>.Empty;
```